### ISSUE TITLE
Query to assess lazy-loading of LCP image

### ISSUE THREAD

**Issue by felixarntz** [GoogleChromeLabs/wpp-research#8]
* Create HTTP Archive query to get **% of WordPress sites that use `loading="lazy"` on LCP image**
    * Relate it with query from #5 for **% of WordPress sites that even have an image as the LCP element**
* Create HTTP Archive query to get **% of WordPress sites that are on version >= 5.5 _and_ have at least one image with `loading="lazy"`**

**Comment by felixarntz** [comment]
cc @vishalkakadiya @pushpakpop

**Comment by felixarntz** [comment]
Also see related #9 and #10

**Comment by vishalkakadiya** [comment]
@felixarntz For the second point here, we can refer to the query below. What do you say?

> Create HTTP Archive query to get % of WordPress sites that are on version >= 5.5 and have at least one image with loading="lazy"

https://discuss.httparchive.org/t/lazy-loaded-lcp-images-by-wordpress-version/2455/2

**Comment by felixarntz** [comment]
Thanks @vishalkakadiya, that's super useful. It actually helps with both questions. I came up with a query based on it that addresses both points. The first part is the most important one anyway.

See #28, ready for your review :)

**Comment by kt-12** [comment]
@felixarntz  does this also address the second point?
`Create HTTP Archive query to get % of WordPress sites on version >= 5.5 and have at least one image with loading="lazy"`. 

I see your query works on the LCP element only, and In the second point, we need to find loding="lazy" across the entire page.  This can be obtained from `_markup` which contains JSON that has `images.img.loading.lazy` that gives the direct count of lazy loaded images on a page.

**Comment by felixarntz** [comment]
@kt-12 Good catch! I should have explained my thinking in the PR, basically I agree with @vishalkakadiya that I don't think we really need this piece of information. I think having the overall number of sites on WP version >= 5.5 is accurate enough, as lazy-loading is enabled by default and it would require a filter to disable it, which is extremely unlikely to be used when looking at scale.
